### ISSUE TITLE
Figure or image 

### ISSUE THREAD

**Issue by ewallace** [swinterbourne/riboviz-work#17]
[Instructions to authors](https://portlandpress.com/biochemist/pages/instructions_to_authors) says:
> Great images can really complement the text of your feature, please include at least one figure or image.

Could we put together some kind of "brainstorm" figure that links together the key concepts we discuss and the way benefits and contributions flow? Or something???

**Comment by ewallace** [comment]
We discussed on 28th Sept. Agreed that two Venn diagrams could do the trick

## Actions

One suggestion is a Venn diagram on actions:
- What the student can do
- What the project needs
- What gets assessed

Or to use colours ( or a gif???) to show what the student can do expanding as the project continues. "The student's capabilities are like a tiny dot that expands."

@ewallace suggests that the expansion of student capabilities is annotated by an arrow pointing from novice to expert?

## Outcomes

Another is a Venn diagram on outcomes:
- What the student gets out
- What the PI/lab gets out
- day-to-day supervisor gets out

@FlicAnderson says "summarize what we are talking about in the article in a visual way"

@ewallace will draft this today in .svg format. Then submit here for approval/edits.

**Comment by FlicAnderson** [comment]
For reference, the program mentioned for dealing with `.svg` files was: [Inkscape](https://inkscape.org/)

**Comment by FlicAnderson** [comment]
Feedback based on initial diagrams draft by Edward: 

Trifoliate Benefits Venn Diagam: 
* perhaps edit 'work here' text? Suggestions? "project work"?

Student Capabilities / Project Needs Diagram: 
* Perhaps make a separate smaller '**student** project needs' circle, to differentiate how this may be smaller than a larger research group project / lab goal, with the aim that the 'student capabilities' should expand to totally fill this 'student project' circle to suggest successful completion :smile: 
* start off with some overlap to initial student capabilities and student project and larger project / lab goal

**Comment by ewallace** [comment]
I’ve uploaded the figure to https://github.com/swinterbourne/riboviz-work/blob/main/open_science_projects_figure_EW_2021-09-28.svg

@FlicAnderson , could I request that sometime tomorrow you include this in the doc with a LaTeX
```
\begin{figure}
…
\end{figure}
```

Then I’ll send out for comments?

**Comment by swinterbourne** [comment]
> I’ve uploaded the figure to https://github.com/swinterbourne/riboviz-work/blob/main/open_science_projects_figure_EW_2021-09-28.svg

It's a bit picky - but the text in figure A has capitalised words (e.g. Student, Day-to-day supervisor) while the text in figure B does not (e.g. novice at start  expert at end), should this be more consistent?

**Comment by FlicAnderson** [comment]
> I’ve uploaded the figure to https://github.com/swinterbourne/riboviz-work/blob/main/open_science_projects_figure_EW_2021-09-28.svg
> 
> @FlicAnderson , could I request that sometime tomorrow you include this in the .docx with a LaTeX
> 
> ```
> \begin{figure}
> …
> \end{figure}
> ```
> 
> Then I’ll send out for comments?

I can have a go at this tomorrow.

**Comment by ewallace** [comment]
> It's a bit picky - but the text in figure A has capitalised words (e.g. Student, Day-to-day supervisor) while the text in figure B does not (e.g. novice at start expert at end), should this be more consistent?

Yes it should. I will fix in the next version. Any other changes needed?

**Comment by ewallace** [comment]
Actually maybe easier to use pandoc markdown image syntax:
https://pandoc.org/MANUAL.html#images

**Comment by FlicAnderson** [comment]
I'm struggling to get the image to show up in the word doc. I've not succeeded so far unfortunately.  Attempt uploaded to branch `add-image-17` at commit  fbb0b5ef7

The alt text shows up in the word doc when the link is in this format in the .md file: 

```
![IMAGE: How to plan for successful student projects](open_science_projects_figure_EW_2021-09-28.svg)
``` 

But I can't seem to figure out why it's not showing up correctly.  

I suggest @ewallace distributes the `main` branch version of the document and the image separately for reviews for now.

**Comment by FlicAnderson** [comment]
Ooh, looks like there's an issue: https://github.com/jgm/pandoc/issues/4058

**Comment by FlicAnderson** [comment]
> Ooh, looks like there's an issue: [jgm/pandoc#4058](https://github.com/jgm/pandoc/issues/4058)

For reference, I've been trying to open this with LibreOffice. It might be that LibreOffice doesn't handle svg well?

**Comment by FlicAnderson** [comment]
> > Ooh, looks like there's an issue: [jgm/pandoc#4058](https://github.com/jgm/pandoc/issues/4058)
> 
> For reference, I've been trying to open this .docx with LibreOffice. It might be that LibreOffice doesn't handle svg well?

This was NOT the case. I tried running the example [here](https://github.com/jgm/pandoc/issues/4058#issuecomment-908942852) and the svg images generated fine. 

I've tried putting the image into a `img` folder, and adjusting the image location, but it doesn't seem to make a difference. (Note: I haven't pushed this change).

**Comment by ewallace** [comment]
> I suggest @ewallace distributes the `main` branch version of the document and the image separately for reviews for now.

Ok, I will do that. Thanks @FlicAnderson!
